### DISCUSSION TITLE
Jupyter Notebook rendering outage

### DISCUSSION THREAD

**Discussion by Zhu-mjl1** [community/community#124790] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

## reporting error
 >"An error occurred Using nbformat v5.10.4 and nbconvert v7.16.1" 
 
 
![IMG_1462](https://github.com/community/community/assets/57784302/c898c186-f39e-46b8-89f9-5f0a0672326f)
when viewing **.ipynb** across repos.

**Comment by CopperEagle** [comment]
Yes, see  #124651. Keep tabs on [gthub status](https://status.github.com). Currently, there is no notice yet.

**Comment by davevad93** [comment]
Hi @Zhu-mjl1 ,

The problem has been fixed a couple of hours ago, you should now be able to see your notebooks: 
- [Incident Report for GitHub](https://www.githubstatus.com/incidents/915nn7rmqth9)

**Comment by mph4** [comment]
Hi Everyone - very sorry for the inconvenience here. We've fixed the problem and are also implementing improvements that we've learned from this to help prevent it (and better respond to any future issues) going forward.
